generator/mavgen.py

```diff
@@ -182,6 +221,9 @@ def update_oneiteration():
                         x.message_target_component_ofs.update(ix.message_target_component_ofs)
                         x.message_names.update(ix.message_names)
                         x.largest_payload = max(x.largest_payload, ix.largest_payload)
+                        if x.enum is not None and ix.enum is not None:
```

Comment:
This probably should be as shown - the enums are always defined as list, so they won't ever be `None`. We should merge if both of these are non-zero-length lists, which think equates to:

```suggestion
                        if x.enum and ix.enum:
```